lib/ui/window/platform_configuration.cc

```diff
@@ -396,4 +429,23 @@ std::string PlatformConfigurationNativeApi::DefaultRouteName() {
       ->DefaultRouteName();
 }
 
+int64_t PlatformConfigurationNativeApi::GetRootIsolateToken() {
+  UIDartState* dart_state = UIDartState::Current();
+  FML_DCHECK(dart_state);
+  return dart_state->GetRootIsolateToken();
+}
+
+void PlatformConfigurationNativeApi::RegisterBackgroundIsolate(
+    int64_t isolate_id) {
```

Comment:
Change the `isolate_id` name to `root_isolate_token` for consistency

(Also applies to other usages such as the `PlatformMessageHandlerStorage` methods)

Reply:
done